derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -253,9 +256,39 @@ pub struct Field {
     setter_attrs: Vec<Attribute>,
 }
 
+/// Options for the field-level `custom` property
+#[derive(Debug, Clone, FromMeta)]
+pub struct CustomField {
+    #[darling(rename = "type")]
+    ty: syn::Type,
+    #[darling(default)]
+    build: Option<BlockContents>,
+}
+
 impl Field {
-    /// Populate `self.field_attrs` and `self.setter_attrs` by draining `self.attrs`
-    fn unnest_attrs(mut self) -> darling::Result<Self> {
+    /// Resolve and check (post-parsing) options which come from multiple darling options
+    ///
+    ///  * Check that we don't have a custom field builder *and* a default value
+    ///  * Populate `self.field_attrs` and `self.setter_attrs` by draining `self.attrs`
+    fn resolve(mut self) -> darling::Result<Self> {
+        // Perhaps at some future point there would be a way to specify the default value of the field in the builder.
+        // That ought possibly to be distinct from the default value of the field in the target.
+        // For now, reject this situation.
```

Comment:
I don't see us ever doing that.

Reply:
Would you like to suggest a different wording for the comment?